Homework8.medium/class.js

```diff
@@ -0,0 +1,63 @@
+class shopingCart{
+    constructor() {
+        this.items = [];
+        this.totalPrice = 0
+        
+    }
+
+
+    additem(product, price) {
+        this.items.push(product)
+        this.totalPrice = this.totalPrice + price;
+         
+    }
+    removeItem(product, price) {
+       this.items.shift(product)
+      this.totalPrice = this.totalPrice - price;}
+
+    
+
+     total(items, totalPrice){
+          this.totalPrice = this.totalPrice;
+          this.items = items.lenght
+
+      }
+
+    }
+
+const cart = new shopingCart();
+cart.additem("steak", 100);
+cart.additem("beer", 50);
+cart.additem("wine",120);
+console.log(cart.items,cart.totalPrice);
+cart.removeItem("wine", 100);
+console.log(cart.items,cart.totalPrice);
```

Comment:
please use the total method of the shopping cart to do this 😄 . The same applies for the other shopping carts